### PR TITLE
Fade transition in groundfloor disabled when coming from network

### PR DESCRIPTION
**Pull request by sisou**
Replaces #90.

### CODE REVIEW

File: src/components/layouts/Groundfloor.vue [hunk 1]
```diff
@@ -25,10 +25,20 @@
...
 
 <script lang="ts">
-import { defineComponent } from '@vue/composition-api';
+import { defineComponent, ref, watch } from '@vue/composition-api';
 
 export default defineComponent({
+    setup(props, context) {
+        const preventNextTransition = ref(false);
 
+        watch(() => context.root.$route, (to, from) => {
```
Comment (onmax) [inline src/components/layouts/Groundfloor.vue]:
```suggestion
        watch(() => context.root.$route, (to) => {
```

File: src/components/layouts/Groundfloor.vue [hunk 2]
```diff
@@ -61,5 +72,9 @@ export default defineComponent({
 
         transition: opacity var(--transition-time) var(--nimiq-ease);
     }
+
+    .settings {
```
Comment (onmax) [inline src/components/layouts/Groundfloor.vue:76]: Maybe this should be better in App.vue, as I see it, there are two problems:
- `.settings` selector is too general and the box in the sidebar with the `settings` class is also getting this class
- The rest of elements have their dimensions in App.vue, not Groundfloor.

**Review by onmax — APPROVED** [review]
It is way cleaner like this, and easier to understand. Thanks for improvising it 😄.

About the changes, I can commit it myself or we can just merge, whatever you prefer

File: src/components/layouts/Groundfloor.vue (hunk 2 above)
Comment (sisou) [reply src/components/layouts/Groundfloor.vue:76]: - The `.settings` selector here is in scoped CSS, so it does not affect the Settings button in the sidebar.
- I tried moving the `.settings` styles from here into `App.vue`, but it somehow got the wrong specificity and ended up preventing the transitions from playing right when going between root and settings.

I will merge this now and release it (as there is another critical fix to be deployed), but feel free to play around with the styles, and commit directly to master if it works.

**Comment by sisou** [comment]
I added you as the co-author of this commit, @onmax.

**Pull request merged**